SwiftyBluetooth/Source/CentralProxy.swift

```diff
@@ -144,7 +144,7 @@ extension CentralProxy {
         let weakRequest = timer.userInfo as! Weak<PeripheralScanRequest>
         
         if weakRequest.value != nil {
-            self.stopScan()
```

Comment:
There is no error here, this is when the scan stop after your specified scan timeout time. 

The scan function timeout time is there to say, "i want to scan for 15 seconds", so when this timer ticks, and your scan request still exist, it means your scan completed without being cancelled by an error or a new scan.